tests/test_duplicate_issue_manager.py

```diff
@@ -198,3 +202,23 @@ def test_get_duplicate_sets(
         expected_duplicate_sets.sort()
         for s1, s2 in zip(duplicate_sets, expected_duplicate_sets):
             assert set(s1) == set(s2)
+
+
+def test_get_hash():
+    img = Image.new("RGB", (200, 200), (255, 0, 0))
+    hash = get_hash(img, {"hash_type": "md5"})
+    assert hash is not None
+
+    hash = get_hash(img, {"hash_type": "whash", "hash_size": 2})
+    assert hash is not None
+
+    hash = get_hash(img, {"hash_type": "phash", "hash_size": 2})
+    assert hash is not None
+
+    # Test calling function with not a real hash
+    try:
```

Comment:
https://www.educative.io/answers/how-to-check-if-an-exception-gets-raised-in-pytest
Check example 2
`import pytest

def func(x):
  if x == 5:
    raise ValueError("x must be a value other than 5")
  return x

def test_func():
  with pytest.raises(ValueError, match="x must be a value other than 5"):
    func(5)

# func(5)     # Raises an exemption

test_func()`